src/transformers/models/bart/modeling_flax_bart.py

```diff
@@ -0,0 +1,1394 @@
+# coding=utf-8
+# Copyright 2021 The Fairseq Authors, The HuggingFace Inc. team and Daniel Stancl. All rights reserved.
```

Comment:
think we sadly have to remove `Daniel Stancl` here :-/ We can't really have individual contributors in the Copyright I think (cc @sgugger )

Reply:
That's okay, removed that! :)